share/vss/vss_test.go

```diff
@@ -0,0 +1,463 @@
+package vss
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/ed25519"
+	"github.com/dedis/crypto/random"
+	"github.com/dedis/crypto/sign"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var suite = ed25519.NewAES128SHA256Ed25519(false)
+
+var reader = random.Stream
+
+var nbVerifiers = 7
+
+var vssThreshold int
+
+var verifiersPub []abstract.Point
+var verifiersSec []abstract.Scalar
+
+var dealerPub abstract.Point
+var dealerSec abstract.Scalar
+
+var secret abstract.Scalar
+
+func genPair() (abstract.Scalar, abstract.Point) {
+	secret := suite.Scalar().Pick(reader)
+	public := suite.Point().Mul(nil, secret)
+	return secret, public
+}
+
+func genCommits(n int) ([]abstract.Scalar, []abstract.Point) {
+	var secrets = make([]abstract.Scalar, n)
```

Comment:
I find more elegent to put `var` in these cases... 
FWIW, https://stackoverflow.com/questions/27919359/why-there-are-two-ways-of-declaring-variables-in-golang-whats-the-different-an